ourney/src/main/java/com/lubycon/ourney/domains/user/service/UserService.java

```diff
@@ -0,0 +1,90 @@
+package com.lubycon.ourney.domains.user.service;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.lubycon.ourney.domains.user.dto.TokenResponse;
+import com.lubycon.ourney.domains.user.entity.User;
+import com.lubycon.ourney.domains.user.entity.UserRepository;
+import com.lubycon.ourney.domains.user.dto.LoginRequest;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.io.*;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+@RequiredArgsConstructor
+@Service
+public class UserService {
+    @Autowired
+    private JwtUtil jwtUtil;
+    @Autowired
+    private UserRepository userRepository;
+    public TokenResponse login(LoginRequest request) {
+        if (!userRepository.findByEmail(request.getEmail()).isEmpty()) { // 이미 가입된 회원 -> 바로 토큰 전달
+            Long userId = userRepository.findIdByKakaoId(request.getKakaoId());
+            String token = jwtUtil.createToken(userId);
+            String refreshToken = jwtUtil.createRefreshToken(userId);
+            return new TokenResponse(userId, token, refreshToken);
+        }
+        else { // 신규 회원 가입 -> 입력 값 저장 후 토큰 전달
+            User user = User.builder()
+                    .kakaoId(request.getKakaoId())
+                    .name(request.getName())
+                    .email(request.getEmail())
+                    .profileImg(request.getProfileImg())
+                    .build();
+            userRepository.save(user);
+
+            Long userId = userRepository.findIdByKakaoId(request.getKakaoId());
+            String token = jwtUtil.createToken(userId);
+            String refreshToken = jwtUtil.createRefreshToken(userId);
+            return new TokenResponse(userId, token, refreshToken);
+        }
+    }
+
+    public LoginRequest getUserInfo(String accessToken) {
+        String reqURL = "https://kapi.kakao.com/v2/user/me";
+        Long   kakaoId = 0L;
```

Comment:
```suggestion
        Long kakaoId = 0L;
```
증맬 마이너한 서제스천임니다 ... ㅎㅎ